homeassistant/components/advantage_air/cover.py

```diff
@@ -0,0 +1,144 @@
+"""Cover platform for Advantage Air integration."""
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    DEVICE_CLASS_DAMPER,
+    SUPPORT_CLOSE,
+    SUPPORT_OPEN,
+    SUPPORT_SET_POSITION,
+    CoverEntity,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import ADVANTAGE_AIR_STATE_CLOSE, ADVANTAGE_AIR_STATE_OPEN, DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up AdvantageAir cover platform."""
+
+    instance = hass.data[DOMAIN][config_entry.entry_id]
+
+    entities = []
+    for ac_key in instance["coordinator"].data["aircons"]:
+        for zone_key in instance["coordinator"].data["aircons"][ac_key]["zones"]:
+            # Only add zone vent controls when zone in vent control mode.
+            if (
+                instance["coordinator"].data["aircons"][ac_key]["zones"][zone_key][
```

Comment:
Depending on review this might be an `if not eval` from an `if eval == 0`

Reply:
So I am explicitly checking for a value of zero from the API, because I wouldn't want this to be true on a different false value like None (which should never happen).

Reply:
👍